src/fluree/db/query/json_ld/response.cljc

```diff
@@ -150,12 +135,18 @@
                                             {id-key c-iri}))
                                         (flake/o f))]
                               (recur r (conj acc res)))
-                            (if (= 1 (count acc))
+                            (cond
+                              (#{:list :set} (-> context (get p-iri) :container))
```

Comment:
This is a very high-hit (performance sensitive) part of the code. I'd prefer this be written as:
```
 (if (and (= 1 (count acc))
          (not (#{:list :set} (-> context (get p-iri) :container)))
   (first acc) 
   acc)
```
Doing two map "get" operations then a set comparison every time is not cheap in the context of possibly millions of times for a single query... at least with the above we are only doing that check when necessary.